CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 1. [#5348](https://github.com/influxdata/chronograf/pull/5348): Add query parameter for dashboard auto refresh
 1. [#5352](https://github.com/influxdata/chronograf/pull/5352): Add etcd as an alternate backend store
 1. [#5367](https://github.com/influxdata/chronograf/pull/5367): Template variables can now select their source database
+1. [#5308](https://github.com/influxdata/chronograf/pull/5308): The hosts page can be disabled by setting the new environment variable `HOST_PAGE_DISABLED` to `true`
```

Comment:
This technically can be set with the command line flags as well. `chronograf -H` or `chronograf --host-page-disabled`

Reply:
good call, added!